### PR TITLE
Unity SDK release 1.0.0

### PR DESCRIPTION
**Pull request by vvoicehovics**
1.0.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Optimove-SDK-Unity
```
Comment (cgwyllie) [inline README.md:1]: Worth a note or table about supported unity versions?

File: unity-sdk/AndroidPlugin/OptimoveUnityPlugin/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <application android:allowBackup="true"
+        android:supportsRtl="true">
+
+        <provider android:authorities="com.optimove.unity.plugin.optimoveinitprovider"
+            android:exported="false"
+            android:name=".OptimoveInitProvider" />
+
+        <!-- Optimove FCM handler -->
+        <service android:name="com.optimove.android.optimobile.FirebaseMessagingService" android:exported="false">
+            <intent-filter
```
Comment (cgwyllie) [inline unity-sdk/AndroidPlugin/OptimoveUnityPlugin/src/main/AndroidManifest.xml]: Still necessary?

Comment (cgwyllie) [reply unity-sdk/AndroidPlugin/OptimoveUnityPlugin/src/main/AndroidManifest.xml]: (sorry, priority)

File: unity-sdk/Assets/Plugins/iOS/OptimoveSDKPlugin.swift [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+    }
+
+    @objc(signOutUser)
+    static func signOutUser() {
+        Optimove.signOutUser()
+    }
+
+    // ========================== MESSAGING ==========================
+
+    @objc(pushRegister)
```
Comment (cgwyllie) [inline unity-sdk/Assets/Plugins/iOS/OptimoveSDKPlugin.swift]: Did the C# API get lined up w/ new naming convention? (pushRequest...)

Comment (cgwyllie) [reply unity-sdk/Assets/Plugins/iOS/OptimoveSDKPlugin.swift]: (guess applies to c# layer + docs only?)

File: README.md (hunk 1 above)
Comment (vvoicehovics) [reply README.md:1]: so far we support >= 2019.3 and only have 1 sdk version -- added a note

File: unity-sdk/AndroidPlugin/OptimoveUnityPlugin/src/main/AndroidManifest.xml (hunk 2 above)
Comment (vvoicehovics) [reply unity-sdk/AndroidPlugin/OptimoveUnityPlugin/src/main/AndroidManifest.xml]: removed

**Pull request merged**